modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -58,6 +58,518 @@ static inline DST add(SRC1 x, SRC2 y)
     return saturate<DST>(x + y, roundf);
 }
 
+enum Arithm { ARITHM_ABSDIFF, ARITHM_ADD, ARITHM_SUBTRACT, ARITHM_MULTIPLY, ARITHM_DIVIDE };
```

Comment:
Why it was moved here ? - it seems that it's usage is not changed

Reply:
Done.